### PR TITLE
Add more tests for calculateRank

### PR DESCRIPTION
**Pull request by nrchandan**
Also, change let to const for API response.

### CODE REVIEW

File: tests/calculateRank.test.js [hunk 1]
```diff
@@ -16,3 +16,35 @@ describe("Test calculateRank", () => {
...
+      calculateRank({
+        totalCommits: 0,
+        totalRepos: 0,
+        followers: 0,
+        contributions: 0,
+        stargazers: 0,
+        prs: 0,
+        issues: 0,
+      })
+    ).toStrictEqual({ level: "A+", score: 51.03597635931825 });
```
Comment (MindLabor) [inline tests/calculateRank.test.js:48]: I have seen that there are other ranks like B+ and C. This test means that the lowest rank is A+ which was not intended, I think.

Comment (nrchandan) [reply tests/calculateRank.test.js:48]: True, looks like `calculateRank` is broken. Will wait for input from @anuraghazra

**Comment by Plavit** [comment]
Still an issue

File: tests/calculateRank.test.js (hunk 1 above)
Comment (Plavit) [reply tests/calculateRank.test.js:48]: Would it maybe make sense to propose the score to be "0" in this case, with the lowest possible rank (C)?

**Review by Plavit — COMMENTED** [review]
The rank calculation appears broken, these tests might provide a good incentive to change and improve it

**Pull request closed without merging**